### PR TITLE
Update JVM debugging docs

### PR DESCRIPTION
**Pull request by dominikschubert**
This new debugging setup is now compatible with both the legacy and the new provider as well as different Java runtimes.

### CODE REVIEW

File: content/en/user-guide/tools/lambda-tools/debugging/index.md [hunk 1]
```diff
@@ -203,12 +202,26 @@ services:
...
-attached to the debugger server. If you want to change that, simply switch to `suspend=n`.
+When creating your Lambda function, set the `_JAVA_OPTIONS` environment variable like so:
+
+{{< command >}}
+$ awslocal lambda create-function --function-name debugfunc \
+--code file://handler.zip \
+--handler myindex.handler \
+--runtime nodejs16.x \
+--timeout 150 \
+--role arn:aws:iam::000000000000:role/lambda-role
```
Comment (joe4dev) [inline content/en/user-guide/tools/lambda-tools/debugging/index.md:216]: missing trailing `\`

File: content/en/user-guide/tools/lambda-tools/debugging/index.md [hunk 2]
```diff
@@ -221,17 +234,23 @@ while [[ -z $(docker ps | grep :5050) ]]; do sleep 1; done
...
 ![Run/Debug Configurations](inteliji-debugging-jvm-2.png)
 
-Now to debug your lambda function, simply click on the `Debug` icon with
-`Remote JVM on LS Debug` configuration selected, and then invoke your
-lambda function.
+Now to debug your Lambda function, simply click on the `Debug` icon with `Remote JVM on LS Debug` configuration selected, and then invoke your Lambda function.
+
+### Alternative setup for IntelliJ IDEA
+
+The debugger can also act as a server by changing the drop-down "Debugger mode" to "Listen to remote JVM".
```
Comment (joe4dev) [inline content/en/user-guide/tools/lambda-tools/debugging/index.md:247]: This section should also mention that the wait script is not needed here

**Review by joe4dev — CHANGES_REQUESTED** [review]
Much-needed improvements 📈 👏 

👉 The create-function command needs to be adjusted for Java.

🟢 Otherwise, I tested the setup on macOS and can confirm it works for `java8.al2`, `java11`, and `java17`.

File: content/en/user-guide/tools/lambda-tools/debugging/index.md [hunk 3]
```diff
@@ -203,12 +202,26 @@ services:
...
 
-Note the `suspend=y` option here, it will delay code execution until the debugger is
-attached to the debugger server. If you want to change that, simply switch to `suspend=n`.
+When creating your Lambda function, set the `_JAVA_OPTIONS` environment variable like so:
+
+{{< command >}}
+$ awslocal lambda create-function --function-name debugfunc \
+--code file://handler.zip \
+--handler myindex.handler \
+--runtime nodejs16.x \
```
[the remark this answers is not in the document]
Comment (dominikschubert) [reply content/en/user-guide/tools/lambda-tools/debugging/index.md:214]: good catch

**Review by joe4dev — APPROVED** [review]
LGTM now 👍

**Pull request merged**